crates/da-clients/aptos/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "aptos-da-client"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+alloy = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0" }
+c-kzg = "1.0.2"
```

Comment:
as the same as before, follow the convention, bring dependencies to workspace dependencies